programs/btc_spv_api/src/spv_processor.rs

```diff
@@ -163,4 +163,27 @@ mod test {
10b3ac8b1cf75af254d8bcd25ec4d12a5b520c86f67b5a022027cbd4721fb6266ff8d3672223f102ac873c4cf2eb2d9296a33fed3664ebf68a014104aff25466888145adefd4a1c112e5177c9b0af932285986523f929cd11e976d11dec9bbac7e391b31b7e9fa07a24aa7ae100f7ffbb1ef864929973f411086bcb2ffffffff0200ac274f000000001976a9141627a6be6590113c2732336397684479d565a7f388ac007f3e36020000001976a914a3d0bcdd84e99472951cc444358ccebac61b486688ac0000000001000000011d8de29c9296e611320ca394f163641a8778d0e9868315c342b4085ec8d59ab0010000008b483045022100a818131085d4370d021a3ae74fd4f8a18cf1c360b15745ab741d8ae0fa77355d02207c460fba7989c293006546326b7b1aeeb2c1e561c5951fb73fdee580d9988a6b0141043872ee2f87817b3dd607998d9476b86726a82f4c1489c94ee2dd6b7a8473c301547d0f8cae1980ef96b995d7db0eb69e9005dc03ab0f156ee7a5e4d36ff62a15ffffffff0200bfb6b3030000001976a9143379b7e05199f19ac136e77298ebd06ee5b1e5c888ac00222048020000001976a914b57e8eea7de80cf79f947bd63a6faeeb59f1f3cd88ac000000000100000001c783c90af206590a0012b4ce6d249083c485bf96b78c83a62dbfcf541c9c84ba010000008a473044022001123857ff1e3a668fadd9c3acac426e111e8ab30b5c2f12d822d364dbedb06c022001dfc1adb694973795b36227b82f82b93d2c39ebf5f1d5e3d729d632c93451740141044a1cd8f265e0a566dc79cf7ea6daf0c29fc2e6281d97a6c495c86a6a4408f6b09ebdae41c8586e70ebfb01acfccee0b5c2d5e5ed5c39cd3644f9bba306f2cc61ffffffff02404b4c00000000001976a9140c6c6c888d64b60fa7d5707f6eb7359048dba59588acb0c40f3f000000001976a914771ba703ec68903093a4fbe3ec812cda8d23ea0088ac0000000001000000021f51f82586c92c57f1839c6b15367600f4555ac38f16de1f2fdf67c61f636fad010000008b4830450221008981eee0c8a82e58335f0c6a0a0d06bae1b8084ebe3f2451f14c271380c5641902206c26b4c5d4cc86c0fe5d38de661324c92b64fdc77873aeddc798332da4655134014104591f9853eb931b5d084801190d282ab7f5c8abbb299f5749d6b105715866e1767c081e8454e9ea3bf63a19c09b39dc1ff71d93270afa94b062a4984af9ad2dbcffffffffa55e28ebbcf94453ff11acb57b4b427c8e429f3a2e5441d63db2afdcb350fbc8010000008a473044022036431306f32b48e3b892d3dd9aa4deec0f5ea1dbabfc106cbe7b44cdbad83ad3022026a19c9093f946a28daa57df2ac9583dc13519d3c6b852cc3bcd2c479565e0b601410467c6736641546582785ca6f0d4e56c08fe67ee882c7907bf16471c85daf96ef48ce804a10a142175ce7d3be7d3bdcf77c884795441aaad7554a92a1ab47f000affffffff0280969800000000001976a91459d3fad18e55f2ffccb6829e67e77cb0e5b8e4d388ac00943577000000001976a9142cb4491ea630d4d5938f340baad7e31e4942626688ac0000000001000000069d74a193c94fa2d9aa9ed6a5d534defa99fe009cd2785d72c358b17bd36763c0010000008b483045022100d9912fadefc20eeb2c59fd10848367542b4065f9da044841361d6c28280fcfd702205078310c595e9efb65670f82b8b1fc07d498df1d56ba74c2186ea0ee8fbc8c11014104f1a2ed99e816a18a92beb211e312b7d07d0a64b8c55739f6b308240b8eae077d9ce5c616c6c32a50d26c04746a67cc6bce7f3466caf153146e883baeb403bf3affffffffb8cba6398365c2667249c5714cfaf0bf71c62b17874db28c582be377e5a408ed010000008b483045022100b386d0a462c3d84dde82ece6e011917370add71d3ac9bbf1a7f6800803d27f7b022060ce9deeb23b08f1658df36fa2333403e83116cb0803919cfd0776e2641af1f5014104f1a2ed99e816a18a92beb211e312b7d07d0a64b8c55739f6b308240b8eae077d9ce5c616c6c32a50d26c04746a67cc6bce";
```

Comment:
Any luck with `include_bytes!()`?

Reply:
yeah its in the version I pushed earlier